ide-support/revsaveasiosstandalone.livecodescript

```diff
@@ -747,21 +747,98 @@ end revSaveAsMobileDeployer
 
 ################################################################################
 
+local sCFBundleIcons
 private command revCopyMobileNewIcons pSettings, pBaseFolder, pAppBundle   
    revStandaloneProgress "Copying icons..."    
-   repeat for each line tType in ("iphone icon" & return & "retina icon,-114" & return & "iOS 7 retina icon,-120" & return & "iphone 6 plus icon,-180" & return & "iphone X icon,-180" & return & \
-         "ipad icon,-72" & return & "ipad retina icon,-144" & return & "iOS 7 ipad icon,-76" & return & "iOS 7 ipad retina icon,-152" & return & "iPad Pro 12.9 icon,-167")
+   /*
+   The entry for the AppStore icon in the JSON file should have the form:
+   {
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x",
+      "filename" : "iTunesArtwork@2x.png"
+    }
+   */
+   local sIconsList 
+   put  \
+         "appstore icon,ios-marketing,1024x1024,1x,ItunesArtwork@2x.png" & return & \
+         "iphone icon,iphone,57x57,1x,Icon-App-57x57@1x.png" & return & \
+         "retina icon,iphone,57x57,2x,Icon-App-57x57@2x.png" & return & \
+         "iOS 7 retina icon,iphone,60x60,2x,Icon-App-60x60@2x.png" & return & \
+         "iphone 6 plus icon,iphone,60x60,3x,Icon-App-60x60@3x.png" & return & \
+         "iphone X icon,iphone,60x60,3x,Icon-App-60x60@3x.png" & return & \
+         "ipad icon,ipad,72x72,1x,Icon-App-72x72@1x.png" & return & \
+         "ipad retina icon,ipad,72x72,2x,Icon-App-72x72@2x.png" & return & \
+         "iOS 7 ipad icon,ipad,76x76,1x,Icon-App-76x76@1x.png" & return & \
+         "iOS 7 ipad retina icon,ipad,76x76,2x,Icon-App-76x76@2x.png" & return & \
+         "iPad Pro 12.9 icon,ipad,83.5x83.5,2x,Icon-App-83.5x83.5@2x.png" \
+         into sIconsList
+   
+   local tAssetsFolder, tAppIconFolder
+   put tempname() into tAssetsFolder
+   create folder  tAssetsFolder
+   put there is a folder tAssetsFolder
+   put tAssetsFolder & slash & "AppIcon.appiconset" into tAppIconFolder
+   create folder tAppIconFolder
+   put there is a folder tAppIconFolder
```

Comment:
These `put there is a folder tAssetsFolder` and `put there is a folder tAppIconFolder` should probably be removed

Reply:
Oops I forgot to remove them :)